css/css-grid/alignment/grid-baseline-align-cycles-001.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
Nit: I'd have added `<meta charset=utf-8>` but it's not really important anyway.